src/acl/agent.ts

```diff
@@ -110,7 +110,7 @@ export function unstable_getAgentAccessModesAll(
  *
  * Keep in mind that this function will not tell you:
  * - what access the given Agent has through other ACL rules, e.g. public or group-specific permissions.
- * - what access the given Agent has to child Resources, in case the associated Resource is a Container.
+ * - what access the given Agent has to child Resources, in case the associated Resource is a Container (see [[unstable_getAgentDefaultAccessModesOne]] for that).
```

Comment:
Just noticing the plural in this name - i.e. I'd have named it `unstable_getAgentDefaultAccessModeOne`.

Reply:
That's kind of weird, since that sounds like you're just getting a single access mode (we don't have functions that return a single mode - you always get all modes together), because it's not clear what `One` refers to then. So to be extremely consistent, that should be `getAgentDefaultAccessModeAllOne`, which I don't think would help anyone, or `getAgentOneDefaultAccessModeAll()` or some other permutation, but that doesn't really feel like it's clarifying a lot either...

(And as you noted elsewhere, this also applies to `getAgentResourceAccessModes`, `getAgentAccessModes`, `getPublicAccessModes`, `getPublicResourceAccessModes` and `getPublicDefaultAccessModes`.)